sdk/keyvault/keyvault-common/review/keyvault-common.api.md

```diff
@@ -0,0 +1,41 @@
+## API Report File for "@azure/keyvault-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ChallengeCallbacks } from '@azure/core-rest-pipeline';
+
+// @public
+export function createChallengeCallbacks(options?: CreateChallengeCallbacksOptions): ChallengeCallbacks;
+
+// @public
+export interface CreateChallengeCallbacksOptions {
+    disableChallengeResourceVerification?: boolean;
+}
+
+// @public
+export interface ParsedKeyVaultEntityIdentifier {
+    name: string;
+    vaultUrl: string;
+    version?: string;
+}
+
+// @public
+export interface ParsedWWWAuthenticate {
+    authorization?: string;
+    authorization_url?: string;
+    resource?: string;
+    scope?: string;
+    tenantId?: string;
+}
+
+// @public
+export function parseKeyvaultIdentifier(collection: string, identifier: string | undefined): ParsedKeyVaultEntityIdentifier;
+
+// @public
+export function parseWWWAuthenticate(wwwAuthenticate: string): ParsedWWWAuthenticate;
```

Comment:
would this be better?
```suggestion
export function parseWWWAuthenticate(wwwAuthenticateString: string): ParsedWWWAuthenticate;
```

Reply:
or even something like
```suggestion
export function parseWWWAuthenticateHeader(headerValue: string): ParsedWWWAuthenticate;
```

Reply:
I have something even better: we don't need to export this at all. Turns out this was only being exported for testing, but now said tests are in `keyvault-common`, so we don't need it in the API surface.